packages/dmn-js-decision-table/src/features/allowed-values/index.js

```diff
@@ -1,7 +1,9 @@
 import AllowedValuesUpdateBehavior from './behavior/AllowedValuesUpdateBehavior';
 import AllowedValuesEditingProvider from './AllowedValuesEditingProvider';
+import Translate from 'diagram-js/lib/i18n/translate';
```

Comment:
Let's depend on `'translate'` as a named module instead of `diagram-js` implementation. Please change it in other places too.

Reply:
Thank you for your reply, I don't understand this very much. Does it mean that its naming needs to be prefixed with Diagram, just like the ’import DiagramTranslate from ”diagram-js/lib/i18n/“‘ you wrote in the dmn-js-drd project？

Reply:
Simply remove the whole `import` statement and apply the [suggestion below](https://github.com/bpmn-io/dmn-js/pull/434#discussion_r349062442) in other places too. The dependency injection mechanism can handle string names for the modules.